BESMod/Systems/Demand/Building/TEASERThermalZone.mo

```diff
@@ -2,9 +2,9 @@
 model TEASERThermalZone
   "Reduced order building model, coupled with TEASER"
   extends BaseClasses.PartialDemand(
+    hBui=0,
+    ABui=0,
     ARoo=sum(zoneParam.ARoof)/nZones,
```

Comment:
Doesn't the `zoneParam.ARoof` include tilt?

Reply:
Yes, I think so. How does this affect this? Do you mean we have to set it to zero as well?

Reply:
@FWuellhorst sorry, my bad. I thought it would be the area of the room and not the roof. Nevertheless, I do not understand why we average the Roof area per Zone. The annotations says that `ARoo` is the Roof are of the building. Hence, it should be the `sum(zoneParam.ARoof)` ?

Reply:
True, basically the same issue. I will remove the default.